doc/cross-compilation.xml

```diff
@@ -167,6 +167,11 @@
       Because of this, a best-of-both-worlds solution is in the works with no splicing or explicit access of <varname>buildPackages</varname> needed.
       For now, feel free to use either method.
     </para>
+    <note><para>
+      There is also a "backlink" <varname>__targetPackages</varname>, yielding a package set which whose <varname>buildPackages</varname> is the current package set.
```

Comment:
"which whose" is a little awkward... I'm not sure what to use but I think just using one of those words is enough.